templates/base.hcl.j2

```diff
@@ -130,3 +130,14 @@ autopilot {
     server_stabilization_time = "{{ nomad_autopilot_server_stabilization_time }}"
 }
 {% endif %}
+
+{% if nomad_ui | default(False) | bool == True %}
```

Comment:
Can you please document the following vars in the readme?
- nomad_ui
- nomad_ui_label_test
- nomad_ui_label_background_color
- nomad_ui_label_text_color

They also need a default value.

Reply:
Thanks for you feedback @rndmh3ro 
I added the doc in the README and I also fix spelling mistake, nomad_ui_label_test to nomad_ui_label_text

Reply:
Thanks!

We still need a default value for theses variables.
So if we set `nomad_ui` to `true`, but not set `nomad_ui_label_text`, the run will fail.

Reply:
> Thanks!
> 
> We still need a default value for theses variables. So if we set `nomad_ui` to `true`, but not set `nomad_ui_label_text`, the run will fail.

Your wishes are my command

Reply:
Sorry, my request wasn't clear enough.

The defaults hould be set in the `defaults/main`.yml

Reply:
Fine. I did it